### DISCUSSION TITLE
[BUG] Cookie is sent by SwaggerUI but can not be received in server-side

### DISCUSSION THREAD

**Discussion by EhsanAramide** [fastapi/fastapi#11334] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Annotated

from fastapi import Cookie, FastAPI

app = FastAPI()


@app.get("/items/")
async def read_items(ads_id: Annotated[str | None, Cookie()] = None):
    return {"ads_id": ads_id}
```

### Description

In SwaggerUI, cookie parameter is sent, but I can not receive the parameter in server-side.
The code is an example I copied it from FastAPI documentation.

My tools' version:
- Python: 3.11.4
- FastAPI: 0.110.0
- Pydantic: 2.6.4
- Uvicorn: 0.28.0

![image](https://github.com/tiangolo/fastapi/assets/72789505/e0507bb4-ad53-419c-be1b-7ba5d21fbc35)

### Operating System

Linux

### FastAPI Version

0.110.0

### Pydantic Version

2.6.4

### Python Version

3.11.4

**Comment by YuriiMotov** [comment, marked as answer]
It looks like the code you run to make the screenshot was different from the code you mentioned before.

Nevertheless, the code from docs example (https://fastapi.tiangolo.com/tutorial/cookie-params/#import-cookie) doesn't work if you use Swagger to test this endpoint. But it works if you run test:
```
def test_cookie():
    client = TestClient(app, cookies={"ads_id": "ads_track"})
    response = client.get("/items/")
    assert response.status_code == 200
    assert response.json() == {"ads_id": "ads_track"}

```

Looks like a Swagger issue

**Reply by omarcruzpantoja** [reply]
Further details related to why it probably does not work in swagger can be found here:


https://github.com/swagger-api/swagger-js/issues/1163

Also question has been asked before:

https://github.com/tiangolo/fastapi/issues/880

**Reply by EhsanAramide** [reply]
Thank you for your responses guys.
@omarcruzpantoja 
@YuriiMotov
